RCS file: /pack/cvsroots/wxwindows/wxWindows/src/msw/crashrpt.cpp,v
retrieving revision 1.4
diff -u -r1.4 crashrpt.cpp
--- crashrpt.cpp	2003/07/15 23:54:37	1.4
+++ crashrpt.cpp	2003/07/17 14:02:39
@@ -1118,7 +1118,7 @@
                          );
 
         wxStrncat(gs_reportFilename, fname,
-                  WXSIZEOF(gs_reportFilename) - strlen(gs_reportFilename) - 1);
+                  WXSIZEOF(gs_reportFilename) - wxStrlen(gs_reportFilename) - 1);
     }
 
     return true;

 	  	 
